File: java/test-fixtures/baseline/src/test/java/qg/LibTest.java
package qg;

import static org.junit.jupiter.api.Assertions.assertEquals;

import org.junit.jupiter.api.Test;

class LibTest {
  @Test
  void addPositive() {
    assertEquals(5, Lib.add(2, 3));
  }

  @Test
  void addNegative() {
    assertEquals(-2, Lib.add(-1, -1));
  }

  @Test
  void multiplyPositive() {
    assertEquals(12, Lib.multiply(3, 4));
  }

  @Test
  void multiplyZero() {
    assertEquals(0, Lib.multiply(0, 5));
  }
}

File: kotlin/test-fixtures/baseline/src/test/kotlin/qg/LibTest.kt
package qg

import kotlin.test.Test
import kotlin.test.assertEquals

class LibTest {
    @Test
    fun addPositive() {
        assertEquals(5, add(2, 3))
    }

    @Test
    fun addNegative() {
        assertEquals(-2, add(-1, -1))
    }

    @Test
    fun multiplyPositive() {
        assertEquals(12, multiply(3, 4))
    }

    @Test
    fun multiplyZero() {
        assertEquals(0, multiply(0, 5))
    }
}
